vmdb/app/controllers/application_controller/tenancy.rb

```diff
@@ -0,0 +1,19 @@
+module ApplicationController::Tenancy
+  extend ActiveSupport::Concern
+
+  included do
+    set_current_tenant_by_subdomain_or_domain(:tenant)
+  end
+
+  # NOTE: remove when these session vars are removed
+  def set_session_tenant(tenant = current_tenant)
+    session[:customer_name] = tenant.try(:name)
+    session[:vmdb_name]     = tenant.try(:vmdb_name)
+    session[:custom_logo]   = tenant.try(:logo?)
```

Comment:
Can we, in the interim, change the session variable name to something like `:has_custom_logo`...or is that way too invasive?  It just reads so funny.